recipe/build.sh

```diff
@@ -11,7 +11,8 @@ if [[ ! -z "$mpi" && "$mpi" != "nompi" ]]; then
   export TESTPROC=4
   export OMPI_MCA_rmaps_base_oversubscribe=yes
   export OMPI_MCA_btl=self,tcp
-  export OMPI_MCA_plm=isolated
+  # for openmpi5 within containers without SSH
+  export OMPI_MCA_plm_ssh_agent=false
   export OMPI_MCA_rmaps_base_oversubscribe=yes
   export OMPI_MCA_btl_vader_single_copy_mechanism=none
```

Comment:
openmpi 5 changed the names and values ~all of these variables. Fortunately, the openmpi 5 conda-forge package now sets the right ones needed for conda-build, so you can delete all of these OMPI_MCA variables and it should work.

Reply:
Thanks @minrk!